js/scenes/UserProfileCreate/UserProfileCreate.js

```diff
@@ -5,46 +5,59 @@ import { colors, typography } from "../../config/styles.js";
 import PropTypes from "prop-types";
 import NovaButton from "../../components/UI/NovaButton/";
 import NovaImperative from "../../components/UI/NovaImperative";
-import {
-  writeNewUserBio,
-  writeNewUserExp,
-  writeNewCompanyData,
-  writeNewUserEducation,
-  writeNewUserGlobal,
-  writeNewUserName,
-  writeNewUserProjects,
-  writeNewUserQuote,
-  writeNewUserReferences,
-  writeNewUserImage,
-  writeNewUserTitle
-} from "../../api/firebaseHelper";
 const { fontMain } = typography;
-const { black, mediumGrey, nearBlack, green, red } = colors;
+const { nearBlack } = colors;
 
-const UserProfileCreate = ({ userID }) => (
+const UserProfileCreate = ({
+  handleChangeBio,
+  handleChangeExperience,
+  handleChangeEducation,
+  handleChangeGlobal,
+  handleChangeName,
+  handleChangeQuote,
+  handleChangeReferences,
+  handleChangeImage,
+  handleChangeTitle,
+  handleSubmit,
+  bio,
+  experience,
+  education,
+  global,
+  name,
+  quote,
+  references,
+  image,
+  title
+}) => (
   <ScrollView>
     <View>
       <NovaImperative color="black" title="Create Your Profile" />
       <Text style={styles.header}>Your full name</Text>
       <View style={styles.textInputContainer}>
         <TextInput
-          placeholder="Your name"
+          placeholder={"Your name"}
+          value={name}
           multiline={true}
           numberOfLines={1}
           style={styles.input}
           placeholderTextColor={nearBlack}
-          onChangeText={text => writeNewUserName(userID, text)}
+          onChange={e => {
+            handleChangeName(e.nativeEvent.text);
```

Comment:
for all of your onChange methods, all you have to do is onChange={(text) => handleFunction(text)}. Please make the changes in this file.